prow/cmd/checkconfig/main.go

```diff
@@ -1104,3 +1113,120 @@ func validateCluster(cfg *config.Config) error {
 	}
 	return utilerrors.NewAggregate(errs)
 }
+
+func validateAdditionalProwConfigIsInOrgRepoDirectoryStructure(root string, filesystem fs.FS) error {
+	var errs []error
+	errs = append(errs, fs.WalkDir(filesystem, root, func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			errs = append(errs, fmt.Errorf("error when walking: %w", err))
+			return nil
+		}
+		// Kubernetes configmap mounts create symlinks for the configmap keys that point to files prefixed with '..'.
```

Comment:
I think we have this logic in a couple places now - might be worth it to DRY out so we only write the comment once?

Reply:
Yeah but its also four lines of code, having to import them from elsewhere means that we will always end up importing the package that contains them, not really worth it here IMHO.